iota/commands/extended/broadcast_bundle.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+import filters as f
+from iota.filters import Trytes
+
+from iota import TransactionTrytes, TransactionHash
+from iota.commands.core import \
+    BroadcastTransactionsCommand
+from iota.commands.extended.get_bundles import GetBundlesCommand
+from iota.commands import FilterCommand, RequestFilter
+
+__all__ = [
+    'BroadcastBundleCommand',
+]
+
+
+class BroadcastBundleCommand(FilterCommand):
+    """
+    Executes ``broadcastBundle`` extended API command.
+
+    See :py:meth:`iota.api.Iota.broadcast_bundle` for more info.
+    """
+    command = 'broadcastBundle'
+
+    def get_request_filter(self):
+        return BroadcastBundleRequestFilter()
+
+    def get_response_filter(self):
+        # Return value is filtered before hitting us.
+        pass
+
+    def _execute(self, request):
+        # Given tail hash, fetches the bundle from the tangle
+        # and validates it.
+        # Returns List[List[TransactionTrytes]]
+        # (outer list has one item in current implementation)
+        bundle = GetBundlesCommand(self.adapter)(transaction=request['tail_hash'])
+        BroadcastTransactionsCommand(self.adapter)(trytes=bundle[0])
+        return {
+            'trytes': bundle[0],
+        }
+        # Future: Support multiple bundles in getBundlesCommand, and
+        # then broadcastBundle can do that as well.
```

Comment:
❔  This comment will probably get lost pretty quickly; could we create a GitHub issue for it, so that we don't lose track of it?

Reply:
https://github.com/iotaledger/iota.py/issues/250